cmd/gateway/azure/gateway-azure.go

```diff
@@ -553,7 +553,11 @@ func (a *azureObjects) Shutdown(ctx context.Context) error {
 // StorageInfo - Not relevant to Azure backend.
 func (a *azureObjects) StorageInfo(ctx context.Context, _ bool) (si minio.StorageInfo, _ []error) {
 	si.Backend.Type = minio.BackendGateway
-	si.Backend.GatewayOnline = minio.IsBackendOnline(ctx, a.httpClient, a.endpoint)
+	host := a.endpoint.Host
+	if a.endpoint.Port() == "" {
+		host = a.endpoint.Host + ":" + a.endpoint.Scheme
```

Comment:
Not sure why Scheme here

Reply:
net.Dial needs to have port always @vadmeste

Reply:
> net.Dial needs to have port always @vadmeste

yeah but a.endpoint.Scheme contains 'http'/'https' and not 80/443

Reply:
yes Go supports :http --> 80 and :https -> :443 @vadmeste